app/src/helpers/requests/documentSearchResults.ts

```diff
@@ -0,0 +1,28 @@
+import { SearchResult } from '../../types/generic/searchResult';
+
+import axios from 'axios';
+
+type Args = {
+    nhsNumber: string;
+    baseUrl: string;
+};
+
+type GetDocumentSearchResultsResponse = {
+    data: Array<SearchResult>;
+};
+
+const getDocumentSearchResults = async ({ nhsNumber, baseUrl }: Args) => {
+    const gatewayUrl = baseUrl + '/SearchDocumentReferences';
+
+    const { data }: GetDocumentSearchResultsResponse = await axios.get(gatewayUrl, {
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        params: {
+            patientId: nhsNumber,
+        },
+    });
+    return data;
```

Comment:
Can we wrap this entire method in a Try Catch too, but in the catch just throw the error upwards where we also catch in the page. It's just good practise for method re-usability